distro/runtime/concierge/smarthome.xargs

```diff
@@ -3,7 +3,7 @@
 # Version configuration
 -Dconcierge.version=5.0.0
 -Djetty.version=9.3.10
--Desh.version=0.9.0.b1
+-Desh.version=0.9.0-SNAPSHOT
```

Comment:
Revert to b1 for now and I will ping you as soon as there is a b2 (planned for December).